packages/core/EventManager.ts

```diff
@@ -92,13 +100,14 @@ export default class EventManager {
   calendarCredentials: CredentialPayload[];
   videoCredentials: CredentialPayload[];
   crmCredentials: CredentialPayload[];
+  appOptions?: AppOptions;
 
   /**
    * Takes an array of credentials and initializes a new instance of the EventManager.
    *
    * @param user
    */
-  constructor(user: EventManagerUser) {
+  constructor(user: EventManagerUser, eventTypeAppMetadata?: z.infer<typeof EventTypeAppMetadataSchema>) {
```

Comment:
If the event type app metadata is passed to the event manager, it will now get the specific app options.